### ISSUE TITLE
Support animated WebP

### ISSUE THREAD

**Issue by mathiasbynens** [dchest/webp-quicklook#1]
This boils down to upgrading to the latest `libwebp`.

Example animated WebP image to test on: https://08334865990100544955.googlegroups.com/attach/8c6f234c4ae89d2d/Zipper_animated.webp

**Comment by dchest** [comment]
Looks like we'll need more than just a libwebp upgrade. I'm not sure how to handle animation with [CGBitmapContextCreate](https://github.com/dchest/webp-quicklook/blob/master/WebPImage.m#L38), so I'll need to learn about it first :-) Any help is appreciated.

**Comment by mathiasbynens** [comment]
No experience in that area either :( Here’s the source code for an old Quick Look plugin for animated GIFs (nowadays OS X supports those by default), does that help? http://sourceforge.net/p/animgifqlgen/code/HEAD/tree/

**Comment by dchest** [comment]
Thanks for the help, but it looks like the linked plugin just renders the GIF with WebKit, which we can't do. Anyway, I'll look into how to implement this. Thank you!

**Comment by dchest** [comment]
Nope, haven't had time to look into it, sorry. Pull requests are welcomed :-)
